analysisservice/src/main/java/nu/yona/server/AnalysisServiceApplication.java

```diff
@@ -51,6 +51,6 @@ public Docket yonaApi()
 	@Bean
 	public CacheManager cacheManager()
 	{
-		return new ConcurrentMapCacheManager("goalConflictMessages");
+		return new ConcurrentMapCacheManager("goalConflictMessages", "usersAnonymized");
```

Comment:
Can I achieve this by just moving the method to `CoreConfiguration` and using `@Autowired
	CacheManager cacheManager;` here combined with the `@EnableCaching` directive?